dart-sdk/lib/src/rust/api.dart

```diff
@@ -0,0 +1,45 @@
+// This file is automatically generated, so please do not edit it.
```

Comment:
The auto-generated files are generated in `build.rs` during dart's build hook. Was it not working as expected?

If not, I would prefer adding a generate command before we push release vs. committing auto-generated files

Reply:
> The auto-generated files are generated in build.rs during dart's build hook. Was it not working as expected?

No, `flutter run` would just fail complaining those files were missing. Maybe that was before the fix you put in for the hooks, I can try again to see if that was it